### PR TITLE
Fix the 'failed to get cluster' dialog issue

### PR DESCRIPTION
**Pull request by wezhang**
Signed-off-by: Wei Zhang <wezhang@outlook.com>

### CODE REVIEW

**Comment by wezhang** [comment]
would fix #1201

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanel.java [hunk 1]
```diff
@@ -261,7 +264,15 @@ public void propertyChange(PropertyChangeEvent evt) {
...
-        errorMessageLabels[ErrorMessageLabelTag.ClusterName.ordinal()] = new JLabel("Cluster Name Should not be null");
+        boolean isSignedIn = false;
+
+        try {
+            isSignedIn = AuthMethodManager.getInstance().isSignedIn();
+        } catch (IOException ignored) { }
+
+        errorMessageLabels[ErrorMessageLabelTag.ClusterName.ordinal()] = new JLabel( isSignedIn ?
+                        "Cluster Name Should not be null, please choose one for submission" :
+                        "Can't list cluster, please login within Azure Explorer and refresh");
```
Comment (djlan) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanel.java]: >Azure Explorer [](start = 65, length = 14)

"View -> Tool Windows -> Azure Explorer"

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/ClusterManagerEx.java [hunk 2]
```diff
@@ -155,26 +158,47 @@ public boolean isLIstAdditionalClusterSuccess() {
...
-        } catch (Exception ex) {
-            DefaultLoader.getUIHelper().showError("Failed to get HDInsight Clusters","List HDInsight Cluster Error");
-        }
+        Optional<List<SubscriptionDetail>> subscriptionList = manager.map(AzureManager::getSubscriptionManager)
+                .flatMap(subscriptionManager -> {
+                    try {
+                        return Optional.ofNullable(subscriptionManager.getSubscriptionDetails());
+                    } catch (IOException e) {
+                        DefaultLoader.getUIHelper().showError("Failed to get HDInsight Clusters. " +
+                                        "Please check your subscription and login at Azure Explorer.",
```
Comment (djlan) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/ClusterManagerEx.java]: >Azure Explorer [](start = 85, length = 14)

"View -> Tool Windows -> Azure Explorer"

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanel.java (hunk 1 above)
Comment (wezhang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanel.java]: fix it

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/ClusterManagerEx.java (hunk 2 above)
Comment (wezhang) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/ClusterManagerEx.java]: fix it

**Pull request merged**